hooks/useUser.ts

```
@@ -12,12 +12,16 @@ export function useUser() {
 
   useEffect(() => {
     if (isLoaded && clerkUser) {
+      // Get role from Clerk publicMetadata, default to 'user' if not set
+      const role = clerkUser.publicMetadata?.role as string || 'user'
```

ESLint

[error] Replace `clerkUser.publicMetadata?.role·as·string` with `(clerkUser.publicMetadata?.role·as·string)`

(prettier/prettier)